proxmox/datadog_checks/proxmox/check.py

```diff
@@ -54,6 +60,49 @@ def _get_vm_hostname(self, vm_id, vm_name, node):
         hostname = hostname_json.get("data", {}).get("result", {}).get("host-name", vm_name)
         return hostname
 
+    def _event_for_task(self, task, node_name):
+        task_type = task.get('type')
+        status = "success" if task.get("status") == "OK" else "error"
+        id = task.get('id') if task.get('id') else node_name
+        user = task.get('user')
+        event_title = EVENT_TYPE_TO_TITLE.get(task_type, task_type)
+        resource_type = resource_type_for_event_type(task_type)
+        resource_id = f'{resource_type}/{id}'
+        self.log.debug(
+            "Creating event for task type: %s ID: %s, resource id %s on node %s",
+            task_type,
+            id,
+            resource_id,
+            node_name,
+        )
+
+        resource = self.all_resources.get(resource_id, {})
+
+        tags = copy.deepcopy(resource.get('tags', []))
```

Comment:
```suggestion
        tags = list(resource.get('tags', []))
```
Assuming tags are always a flat array, we don't need `copy.deepcopy` to copy an array. We can get rid of that `copy` import.

Reply:
Good idea, updated!